plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -316,6 +348,208 @@ function format(packageFolder) {
   return 0;
 }
 
+/**
+ * Use `npm outdated` to find which dependencies are not up to date.
+ *
+ * @returns a dict keyed by package name
+ *
+ * @see https://docs.npmjs.com/cli/v8/commands/npm-outdated
+ *
+ * #### Example
+ * ```js
+ *  {
+ *    "@kinvolk/headlamp-plugin": {
+ *      "current": "0.5.0",
+ *      "wanted": "0.5.1",
+ *      "latest": "0.5.1",
+ *      "dependent": "pod-counter",
+ *      "location": "/home/rene/dev/headlamp/plugins/examples/pod-counter/node_modules/@kinvolk/headlamp-plugin"
+ *    }
+ *  }
+ * ```
+ *
+ * #### Example: Nothing needs updating?
+ * ```js
+ *  {}
+ * ```
+ */
+function getNpmOutdated() {
+  let result = null;
+  try {
+    result = child_process.execSync('npm outdated --json', {
+      encoding: 'utf8',
+    });
+    // console.log(result);
```

Comment:
Remove the comment, or make it print a meaningful message.

Reply:
Thanks. I removed it.